java-persistence/src/main/java/com/whitecloak/training/inventory/service/ItemService.java

```diff
@@ -2,8 +2,10 @@
 
 import com.whitecloak.training.inventory.controller.request.ItemForm;
 import com.whitecloak.training.inventory.controller.response.ItemResource;
+import com.whitecloak.training.inventory.persistence.entity.ItemEntity;
 
 public interface ItemService {
 
     ItemResource create(ItemForm form);
+    ItemEntity getItem(Long id);
```

Comment:
Entity has leaked outside the gateways. Remember to contain entities only in the gateways.